configure.ac

```diff
@@ -28,7 +28,12 @@ AC_MSG_NOTICE([Configuring OCaml version AC_PACKAGE_VERSION])
 # Configuration variables
 
 ## Command-line arguments passed to configure
-CONFIGURE_ARGS="$*"
+CONFIGURE_ARGS=''
+for arg in "$@" ; do
+  CONFIGURE_ARGS="$CONFIGURE_ARGS${CONFIGURE_ARGS:+ }'$arg'"
+done
+# Escape CONFIGURE_ARGS for including in a Makefile
+CONFIGURE_ARGS=$(echo "$CONFIGURE_ARGS" | sed -e 's/\$/$$/g;s/#/\\#/g')
```

Comment:
Again, is this departing from the automake behavior?

Reply:
This is nothing to do with automake - our build system includes a utility target `make reconfigure` which is supposed to invoke configure with the same arguments it was originally run with... it was broken!